Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1017,8 +1017,7 @@ void TextureCache::CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFormat
 		entry->hash = GetHash64(dst, (int)entry->size_in_bytes, g_ActiveConfig.iSafeTextureCache_ColorSamples);
 
 		// Invalidate all textures that overlap the range of our texture
-		TexCache::iterator
-			iter = textures_by_address.begin();
+		iter = textures_by_address.begin();
```

Comment:
Please rename the variable, don't repurpose an old one that has different intent.